code/game/objects/items/devices/radio/radio.dm

```diff
@@ -74,6 +74,16 @@
 	/// associative list of the encrypted radio channels this radio can listen/broadcast to, of the form: list(channel name = channel frequency)
 	var/list/secure_radio_connections
 
+	/// overlay when speaker is on
+	var/overlay_speaker_idle = "s_idle"
+	/// overlay when recieving a message
+	var/image/overlay_speaker_active = "s_active"
+
+	/// overlay when mic is on
+	var/overlay_mic_idle = "m_idle"
+	/// overlay when speaking a message (will most likely be displayed at same time as the speaker active, please account for this.)
```

Comment:
What is the status of the last part of your comment? Have you accounted for it? If not you should not leave this in the comment and should instead file a bug report afterwards, or keep it in a non-doc comment.